components/brave_vpn/brave_vpn_service_desktop.h

```diff
@@ -39,7 +40,8 @@ class BraveVpnServiceDesktop
  public:
   BraveVpnServiceDesktop(
       scoped_refptr<network::SharedURLLoaderFactory> url_loader_factory,
-      PrefService* prefs);
+      PrefService* prefs,
+      mojo::PendingRemote<skus::mojom::SkusService> pending_skus_service);
```

Comment:
the convention is just skus_service or skus_service_remote

Reply:
Fixed in https://github.com/brave/brave-core/pull/10847/commits/bc08735f11dc33351fe3845cef9a29959b4e6eb4